codes/classical/q-ary_digits/constant_weight.yml

```diff
@@ -0,0 +1,31 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+
+# code id, physical, logical are all lower case
+code_id: constant_weight
+physical: q-ary_digits
+logical: q-ary_digits
+
+name: 'Constant Weight Codes'
+
+description: 'Constant weight codes, also known as m-of-n codes, are codes, usually binary, of length \(n\) with minimum distance \(d\) in which all codewords have a constant Hamming weight of \(m\). The maximum number of codewords in a constant weight code is denoted \(A(n,d,m)\) and the bounds on this number are currently a topic of research. \cite{doi:10.37236/1162}'
+
+
+protection: 'Binary constant weight codes with weight \(m\) are guaranteed to detect one error, as a flip in any bit will result in a vector with Hamming weight \(m-1\) or \(m+1\).'
```

Comment:
Any code (constant weight or not) corrects (d-1)/2 errors or d-1 erasures. Please revise.